### ISSUE TITLE
Typescript: Incorrectly detected comments inside JSX cause errors

### ISSUE THREAD

**Issue by crohlfs** [prettier/prettier#1990]
```ts
var example1 = <div>
	https://test
</div>;

var example2 = <div>
	/*test*/
</div>;
```

Reproductions
[With //](https://prettier.github.io/prettier/#%7B%22content%22%3A%22var%20error%20%3D%20%3Cdiv%3E%5Cn%5Cthttps%3A%2F%2Ftest%5Cn%3C%2Fdiv%3E%3B%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D)
[With /*](https://prettier.github.io/prettier/#%7B%22content%22%3A%22var%20error%20%3D%20%3Cdiv%3E%5Cn%5Ct%2F*test*%2F%5Cn%3C%2Fdiv%3E%3B%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D)

**Comment by vjeux** [comment]
Thanks!

**Comment by vjeux** [comment]
I just published 1.4.3 which fixes it. Thanks for reporting it!
